go.mod

```diff
@@ -3,7 +3,7 @@ module vitess.io/vitess
 go 1.12
 
 require (
```

Comment:
I'm going to merge upstream master, which has been modularized, after this lands, so you can ignore these for now.

Reply:
ack